widgetastic_manageiq/__init__.py

```diff
@@ -94,6 +95,83 @@ def currently_selected(self):
             return None
 
 
+class SectionedBootstrapSelect(BootstrapSelect):
```

Comment:
Ultimately I think we're going to want another wt.pf-react repo, or a separate module in wt.pf to store these widgets. Its fine here for now though.

Reply:
Agreed, @lina-nikiforova is working on the react version of this widget in #8942, this widget is just a special kind of `BootstrapSelect` however, no react necessary.